node/cli/src/cli.rs

```diff
@@ -58,6 +58,7 @@ pub struct Cli {
     pub no_hardware_benchmarks: bool,
 }
 
+#[allow(clippy::large_enum_variant)]
```

Comment:
fix for
`./scripts/gear.sh clippy gear --all-targets --features=runtime-benchmarks`